pkg/cmd/operator/cmd.go

```diff
@@ -113,6 +120,11 @@ func NewOperator() *cobra.Command {
 			ctx, cancel := context.WithCancel(context.Background())
 			defer cancel()
 
+			caConfigMapPath := filepath.Join(caConfigMapMountPath, caConfigMapName)
+			if _, err := os.Stat(caConfigMapPath); err == nil {
```

Comment:
also i would like us to comment why we are checking for file to exist to start the reactor?